src/sessions/p2p_session.rs

```diff
@@ -17,6 +18,7 @@ use std::convert::TryInto;
 const RECOMMENDATION_INTERVAL: Frame = 60;
 const MIN_RECOMMENDATION: u32 = 3;
 const MAX_EVENT_QUEUE_SIZE: usize = 100;
+const MAX_CHECKSUM_HISTORY_SIZE: usize = 32;
```

Comment:
Is this constant always supposed to be the same as `network::protocol::MAX_CHECKSUM_HISTORY_SIZE` ? They should not be defined twice.